fireplace/utils.py

```diff
@@ -182,6 +183,15 @@ def play_full_game():
 		cards_to_mulligan = random.sample(player.choice.cards, mull_count)
 		player.choice.choose(*cards_to_mulligan)
 
+	try:
+		play_full_game_turn_loop(game)
```

Comment:
It plays all the turns. Shall. Split it so one call plays one turn for each player?